docs/quality-assurance/roles/expert-test-engineer.md

```diff
@@ -0,0 +1,25 @@
+| ​ | ​Design manual tests | ​A feature needs to be manually tested so the tester uses test design techniques to create both happy and unhappy path test plans. |
+| ​ | ​Design Automated tests | ​A feature will need automated tests; the tester uses test design techniques to create the relevant automated tests, using TDD and BDD approaches. |
+| ​Leads Testing (20%) | Leads the team in ensuring that the product is tested to a satisfactory level to give confidence that when released it will function as intended | During a sprint, the tester notices that the developers do not have enough work to do. The tester discusses with the team and proposes that there are some test tasks that they could work on to keep the sprint on track. |
+|   | Leads the use of Behaviour Driven Development (BDD) within the scrum team | The Delivery Manager is finding it challenging to add acceptance criteria to a PBI. They work with the Delivery Manager using BDD techniques to create acceptance criteria. |
+| ​ | Carries out exploratory testing  | A feature is under test; the tester feels that some exploratory testing will help look for defects.  They use a test charter and test persona to carry out some exploratory testing. |
+| ​ | Responsible for coaching less experienced test automation engineers | The tester coaches a junior tester in Behaviour Driven Development, reviewing his work and providing feedback and advice. |
+| ​ | Responsible for ensuring that testing on a project adds up to a coherent whole | A project uses several different testing techniques - automated, manual, non-functional. The tester sees that there is a lack of security testing, so they recommend to the Delivery Manager that several security testing PBI&#39;s are added to the backlog. |
+| ​ | Will be involved in hiring and other resourcing decisions (but is not responsible for recruitment) | Having received appropriate training, the tester is a member of the board on an interview of a test automation engineer. They assist in creating the questions used. |
+| ​Software Testing Subject Matter Expert (10%) | Advise the team on the best approach for automated tests | ​The team are building a new application; the tester reviews the PBI&#39;s and advises on the best approach for automated testing. |
+| ​ | ​Helps the team on using best practice for writing Behaviour Driven Development testing | ​The tester sees that some of the team have less confidence than others in using BDD. They offer to provide some 1-2-1 coaching to improve the overall level of skill within the team. |
+| ​ | ​​Looks at the testability of a product | As part of backlog refinement they see that asking the developers to create a stub will allow the team to be able to progress testing rather than having to wait for another PBI to be completed. |
```

Comment:
I disagree with this point. Surely an Expert Test Engineer would not only be capable of making their own stubs, they would be promoting technologies and techniques for doing so across the organisation?

Reply:
Agree that the expectation would change as they move up the roles. Have changed the wordung to state they would implement not ask. Also changed on the senior.